src/components/LayoutSideNavigation/LayoutSideNavigation.css

```diff
@@ -35,14 +35,14 @@
   padding: 24px;
 
   @media (--viewportLarge) {
-    padding: 116px 15vw 82px 82px;
+    padding: 112px 36px 82px 82px;
     background-color: var(--matterColorLight);
     border-left-width: 1px;
     border-left-style: solid;
     border-left-color: var(--matterColorNegative);
   }
 
-  @media (--viewportXLarge) {
-    padding-right: 25vw;
+  @media (--viewportLargeWithPaddings) {
+    padding: 112px calc((100% - 1056px) / 2 + 36px) 82px 82px;
```

Comment:
If you keep the `+ 36px` after viewportLargeWithPaddings, it will shrink the content smaller than 1056px:
![screenshot 2019-02-06 at 12 10 05](https://user-images.githubusercontent.com/717315/52334393-3c393d80-2a08-11e9-8193-5e290bd67b5e.png)

Reply:
This applies to the right side. On the left side we want to achieve this effect since the content is fixed 200px wide and the reserved space is bigger.